### DISCUSSION TITLE
I dint got my github copilet education kit

### DISCUSSION THREAD

**Discussion by Aashish7112004** [community/community#106408] [GitHub Education]
### Select Topic Area

Question

### Body

![Screenshot 2024-02-04 165346](https://github.com/community/community/assets/158753294/296fab3d-62be-467a-9aca-3bc766d21104)
Congrats! Your request for [@Aashish7112004] was approved and your benefits will be available soon.
i got apporved on feb 3 still waithing for this to use and i am also exiceted to use it
 please github community verify my account and give me copilet development kit

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Aashish7112004 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
